2023-03-01 20:59:51,899 CRITICAL   client_log_config.py Критическая ошибка
2023-03-01 20:59:51,899 ERROR      client_log_config.py Ошибка
2023-03-01 20:59:51,899 DEBUG      client_log_config.py Отладочная информация
2023-03-01 20:59:51,899 INFO       client_log_config.py Информационное сообщение
2023-03-01 21:00:01,598 INFO       client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, порт: 7778
2023-03-01 21:00:42,406 INFO       client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, порт: 7778
2023-03-01 21:00:42,406 DEBUG      utils.py Сформировано presence сообщение для пользователя Guest
2023-03-01 21:00:42,406 DEBUG      utils.py Разбор сообщения от сервера: {'response': 200}
2023-03-01 21:00:42,406 INFO       client.py Принят ответ от сервера 200 : OK
2023-03-04 10:55:37,073 DEBUG      client.py Была вызвана функция create_arg_parser c параметрами (), {}. Вызов из модуля __main__.Вызов из функции main
2023-03-04 10:55:37,073 INFO       client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, порт: 7778
2023-03-04 10:55:37,073 DEBUG      utils.py Сформировано presence сообщение для пользователя Guest
2023-03-04 10:55:37,073 DEBUG      client.py Была вызвана функция create_presence c параметрами (), {}. Вызов из модуля utils.Вызов из функции main
2023-03-04 10:55:37,074 DEBUG      client.py Была вызвана функция send_message c параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 36122), raddr=('127.0.0.1', 7778)>, {'action': 'presence', 'time': 1677912937.0738435, 'user': {'account_name': 'Guest'}}), {}. Вызов из модуля utils.Вызов из функции main
2023-03-04 10:55:37,075 DEBUG      client.py Была вызвана функция receive_message c параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 36122), raddr=('127.0.0.1', 7778)>,), {}. Вызов из модуля utils.Вызов из функции main
2023-03-04 10:55:37,075 DEBUG      utils.py Разбор сообщения от сервера: {'response': 200}
2023-03-04 10:55:37,075 INFO       client.py Принят ответ от сервера 200 : OK
